src/Settings.fs

```diff
@@ -28,13 +27,6 @@ module Settings =
         builder.ConnectionString
 
     let load () =
-        let builder =
```

Comment:
Why have we removed this?

Reply:
Unused variable. The project worked normally without it, do we need this line for something else?

Reply:
We had it to use Microsoft's configuration module, we can add it back with JSON/dotenv support in the future.